docs/machine-learning/tutorials/sentiment-analysis-cli.md

```diff
@@ -79,7 +79,7 @@ We are going to use an existing dataset used for a 'Sentiment Analysis' scenario
 1. Run the following ML.NET CLI command:
 
     ```console
-    mlnet auto-train --task binary-classification --dataset "yelp_labelled.txt" --label-column-index 1 --has-header false --max-exploration-time 10
+    mlnet auto-train --task binary-classification --dataset "yelp_labelled.txt" --label-column-index 1 --has-header false --max-exploration-time 10
```

Comment:
I'm not seeing anything changed here. Not sure why GitHub shows this as if something has changed.